tests/envcheck/envcheck.go

```diff
@@ -85,9 +87,9 @@ type runConfig struct {
 var (
 	// noOverload holds the proto representation of the "no_matching_overload" error.
 	noOverload = &exprpb.ExprValue{Kind: &exprpb.ExprValue_Error{
-		Error: &exprpb.ErrorSet{Errors: []*spb.Status{
+		Error: &exprpb.ErrorSet{/*Errors: []*spb.Status{
 			status.New(codes.InvalidArgument, "no_matching_overload").Proto(),
-		}},
+		}*/},
```

Comment:
Remove comment ... ?  It looks like you want to set the code and message somewhere.

Reply:
Done. I had encountered a strange bazel error and isolated it to the grpc imports. Should be fixed now, just forgot to remove the comments.